yamllint/rules/quoted_strings.py

```diff
@@ -15,15 +15,23 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 """
-Use this rule to forbid any string values that are not quoted.
-You can also enforce the type of the quote used using the ``quote-type`` option
-(``single``, ``double`` or ``any``).
+Use this rule to forbid any string values that are not quoted, or to prevent
+quoted strings without needing it. You can also enforce the type of the quote
+used.
+
+.. rubric:: Options
+
+* ``quote-type`` defines allowed quotes: ``single``, ``double`` or ``any``
+  (default).
+* ``required`` defines whether using quotes in string values is required
+  (``true``, default) or not (``false``), or mandatory when really needed
```

Comment:
`or mandatory when really needed` → `or allowed when really needed`?

Otherwise it could be interpreted the other way? (I may be wrong, I'm not a native English speaker.)

Reply:
I find this one better: `or only allowed when really needed`. Ok?

Reply:
Perfect!